docs/content/concepts/authorization/authorizers.md

```diff
@@ -276,9 +276,24 @@ The webhook will receive JSON-marshalled `SubjectAccessReview` objects, that (co
 }
 ```
 
-
     The extra field will contain the logical cluster _name_ (e.g. o43u2gh528rtfg721rg92), not the human-readable path. Webhooks need to resolve the name to a path themselves if necessary.
 
+### Authorizer order
+
+By default, the authorizers are evaluated in the following order:
+
+1. **Always Allow Groups Authorizer**
+2. **Always Allow Paths Authorizer**
+3. **RBAC Chain**
+4. **Webhook Authorizer**
+
+The order in which authorizers are evaluated can be configured. This allows administrators to customize the authorization flow.
+It can be done by setting `--authorization-order` flag in the kcp server. This flag accepts a comma-separated list of authorizer names, which will be evaluated in the specified order.
+
+```sh
+--authorization-order=Webhook,AlwaysAllowGroups,AlwaysAllowPaths,RBAC
+```
```

Comment:
Can we add a quick sentence above describing that this is an example of how to configure the Webhook to be called first? Just to make clear it's not the default setting of the flag.